### ISSUE TITLE
heap-buffer-overflow in RTCP::FeedbackPsAfbPacket::Parse()

### ISSUE THREAD

**Issue by ibc** [versatica/mediasoup#251]
```
==406==ERROR: AddressSanitizer: heap-buffer-overflow on address 0x6020001eef1f at pc 0x000000952989 bp 0x7fff17be3010 sp 0x7fff17be3008
READ of size 1 at 0x6020001eef1f thread T0
    #0 0x952988 in Utils::Byte::Get4Bytes(unsigned char const*, unsigned long) /mediasoup/worker/out/../include/Utils.hpp:125:20
    #1 0xac1202 in RTC::RTCP::FeedbackPsAfbPacket::Parse(unsigned char const*, unsigned long) /mediasoup/worker/out/../src/RTC/RTCP/FeedbackPsAfb.cpp:32:8
    #2 0xa4dddb in RTC::RTCP::FeedbackPacket<RTC::RTCP::FeedbackPs>::Parse(unsigned char const*, unsigned long) /mediasoup/worker/out/../src/RTC/RTCP/Feedback.cpp:171:15
    #3 0xa2746b in RTC::RTCP::Packet::Parse(unsigned char const*, unsigned long) /mediasoup/worker/out/../src/RTC/RTCP/Packet.cpp:127:17
    #4 0xb4ba60 in fuzz(unsigned char const*, unsigned long) /mediasoup/worker/out/../fuzzer/fuzzer.cpp:75:26
    #5 0xb4bd2a in LLVMFuzzerTestOneInput /mediasoup/worker/out/../fuzzer/fuzzer.cpp:100:2
    #6 0x45f47a in fuzzer::Fuzzer::ExecuteCallback(unsigned char const*, unsigned long) /tmp/final/llvm.src/projects/compiler-rt/lib/fuzzer/FuzzerLoop.cpp:576:15
    #7 0x45eb95 in fuzzer::Fuzzer::RunOne(unsigned char const*, unsigned long, bool, fuzzer::InputInfo*, bool*) /tmp/final/llvm.src/projects/compiler-rt/lib/fuzzer/FuzzerLoop.cpp:485:3
    #8 0x460fcb in fuzzer::Fuzzer::MutateAndTestOne() /tmp/final/llvm.src/projects/compiler-rt/lib/fuzzer/FuzzerLoop.cpp:713:19
    #9 0x461ca5 in fuzzer::Fuzzer::Loop(std::__Fuzzer::vector<std::__Fuzzer::basic_string<char, std::__Fuzzer::char_traits<char>, std::__Fuzzer::allocator<char> >, fuzzer::fuzzer_allocator<std::__Fuzzer::basic_string<char, std::__Fuzzer::char_traits<char>, std::__Fuzzer::allocator<char> > > > const&) /tmp/final/llvm.src/projects/compiler-rt/lib/fuzzer/FuzzerLoop.cpp:844:5
    #10 0x4584f3 in fuzzer::FuzzerDriver(int*, char***, int (*)(unsigned char const*, unsigned long)) /tmp/final/llvm.src/projects/compiler-rt/lib/fuzzer/FuzzerDriver.cpp:765:6
    #11 0x4801d2 in main /tmp/final/llvm.src/projects/compiler-rt/lib/fuzzer/FuzzerMain.cpp:20:10
    #12 0x7fd2a3c2982f in __libc_start_main (/lib/x86_64-linux-gnu/libc.so.6+0x2082f)
    #13 0x434a28 in _start (/mediasoup/worker/out/Release/mediasoup-worker-fuzzer+0x434a28)

0x6020001eef1f is located 3 bytes to the right of 12-byte region [0x6020001eef10,0x6020001eef1c)
allocated by thread T0 here:
    #0 0x5951b2 in operator new[](unsigned long) /tmp/final/llvm.src/projects/compiler-rt/lib/asan/asan_new_delete.cc:109:3
    #1 0x45f361 in fuzzer::Fuzzer::ExecuteCallback(unsigned char const*, unsigned long) /tmp/final/llvm.src/projects/compiler-rt/lib/fuzzer/FuzzerLoop.cpp:563:23
    #2 0x45eb95 in fuzzer::Fuzzer::RunOne(unsigned char const*, unsigned long, bool, fuzzer::InputInfo*, bool*) /tmp/final/llvm.src/projects/compiler-rt/lib/fuzzer/FuzzerLoop.cpp:485:3
    #3 0x460fcb in fuzzer::Fuzzer::MutateAndTestOne() /tmp/final/llvm.src/projects/compiler-rt/lib/fuzzer/FuzzerLoop.cpp:713:19
    #4 0x461ca5 in fuzzer::Fuzzer::Loop(std::__Fuzzer::vector<std::__Fuzzer::basic_string<char, std::__Fuzzer::char_traits<char>, std::__Fuzzer::allocator<char> >, fuzzer::fuzzer_allocator<std::__Fuzzer::basic_string<char, std::__Fuzzer::char_traits<char>, std::__Fuzzer::allocator<char> > > > const&) /tmp/final/llvm.src/projects/compiler-rt/lib/fuzzer/FuzzerLoop.cpp:844:5
    #5 0x4584f3 in fuzzer::FuzzerDriver(int*, char***, int (*)(unsigned char const*, unsigned long)) /tmp/final/llvm.src/projects/compiler-rt/lib/fuzzer/FuzzerDriver.cpp:765:6
    #6 0x4801d2 in main /tmp/final/llvm.src/projects/compiler-rt/lib/fuzzer/FuzzerMain.cpp:20:10
    #7 0x7fd2a3c2982f in __libc_start_main (/lib/x86_64-linux-gnu/libc.so.6+0x2082f)

SUMMARY: AddressSanitizer: heap-buffer-overflow /mediasoup/worker/out/../include/Utils.hpp:125:20 in Utils::Byte::Get4Bytes(unsigned char const*, unsigned long)
Shadow bytes around the buggy address:
  0x0c0480035d90: fa fa fd fd fa fa fd fd fa fa fd fd fa fa fd fd
  0x0c0480035da0: fa fa fd fd fa fa fd fd fa fa fd fd fa fa fd fd
  0x0c0480035db0: fa fa fd fd fa fa fd fa fa fa fd fd fa fa fd fa
  0x0c0480035dc0: fa fa fd fd fa fa fd fa fa fa fd fd fa fa fd fa
  0x0c0480035dd0: fa fa fd fd fa fa fd fa fa fa fd fd fa fa fd fd
=>0x0c0480035de0: fa fa 00[04]fa fa fa fa fa fa fa fa fa fa fa fa
  0x0c0480035df0: fa fa fa fa fa fa fa fa fa fa fa fa fa fa fa fa
  0x0c0480035e00: fa fa fa fa fa fa fa fa fa fa fa fa fa fa fa fa
  0x0c0480035e10: fa fa fa fa fa fa fa fa fa fa fa fa fa fa fa fa
  0x0c0480035e20: fa fa fa fa fa fa fa fa fa fa fa fa fa fa fa fa
  0x0c0480035e30: fa fa fa fa fa fa fa fa fa fa fa fa fa fa fa fa
Shadow byte legend (one shadow byte represents 8 application bytes):
  Addressable:           00
  Partially addressable: 01 02 03 04 05 06 07
  Heap left redzone:       fa
  Freed heap region:       fd
  Stack left redzone:      f1
  Stack mid redzone:       f2
  Stack right redzone:     f3
  Stack after return:      f5
  Stack use after scope:   f8
  Global redzone:          f9
  Global init order:       f6
  Poisoned by user:        f7
  Container overflow:      fc
  Array cookie:            ac
  Intra object redzone:    bb
  ASan internal:           fe
  Left alloca redzone:     ca
  Right alloca redzone:    cb
  Shadow gap:              cc
==406==ABORTING
MS: 1 ChangeBit-; base unit: b0b3b3c63cbeaf530e0be5c084aa48110b1e6a27
0x8f,0xce,0x0,0x2,0x83,0x88,0xcd,0x0,0x0,0xc,0x0,0x0,
\x8f\xce\x00\x02\x83\x88\xcd\x00\x00\x0c\x00\x00
artifact_prefix='fuzzer/reports/'; Test unit written to fuzzer/reports/crash-db6919ea906ac9f5d71a520dc342932b02559a42
Base64: j84AAoOIzQAADAAA
```

**Comment by ibc** [comment]
`worker/fuzzer/reports/leak-429569dcc2dec38f982e2ea5ac0f7eba6c5a4ba4` pushed.

**Comment by ibc** [comment]
yes, it was fixed in 8ff4b23b575a03f1f934da7f6e8b97c5124d2e14.
